cordova-plugin-moodleapp/src/ts/plugins/SecureStorage.ts

```diff
@@ -0,0 +1,76 @@
+// (C) Copyright 2015 Moodle Pty Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Allows retrieving and storing items in a secure storage.
+ */
+export class SecureStorage {
+
+    /**
+     * Get one or more values.
+     *
+     * @param names Names of the values to get.
+     * @param collection The collection where the values are stored.
+     * @returns Object with name -> value.
+     */
+    async get(names: string | string[], collection: string): Promise<Record<string, string | null>> {
```

Comment:
```suggestion
    async get(names: string | string[], collection: string): Promise<Record<string, string>> {
```

It seems like both Android and iOS already filter out null values, so it should be fine to type it without null.